packages/appium/docs/overrides/partials/footer.html

```diff
@@ -0,0 +1,87 @@
+<!-- Sponsor thanks -->
+
+<div class="appium-sponsor-thanks">
+  <h3>With thanks to our Development Partner</h3>
+  <div style="text-align: center">
+    <a href="https://www.headspin.io/solutions/appium-mobile-test-automation">
+      <img src="/docs/en/assets/images/sponsor-logo-headspin.png" style="width: 200px;" />
+    </a>
+  </div>
+</div>
+
+<!-- Footer -->
```

Comment:
note that to create a custom footer, i had to create an override file and copy in the existing default footer code to augment it.